commit a227e7ad566c8b969b5a1848b5578dcd7c349a52
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2019-08-29T22:35:41+00:00 (6 weeks ago)
    
    joyent/bugview#37 illumos-live links should be rewritten to point at smartos-live
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/jirapub.js b/jirapub.js
index 740e206..9a8adee 100644
--- a/jirapub.js
+++ b/jirapub.js
@@ -569,6 +569,11 @@ fix_url(input)
 				m: '/smartos-live',
 				p: '/joyent/smartos-live'
 			},
+			{
+				h: 'github.com',
+				m: '/illumos-live',
+				p: '/joyent/smartos-live'
+			},
 			{
 				h: 'github.com',
 				m: '/illumos-extra',
